packages/node/src/utils.ts

```diff
@@ -3,6 +3,22 @@ import { pathToRegexp } from 'path-to-regexp';
 import type { IncomingMessage } from 'http';
 import { extname } from 'path';
 
+// When exiting this process, wait for Vercel Function server to finish
+// all its work, especially waitUntil promises before exiting this process.
+//
+// Here we use a short timeout (10 seconds) to let the user know that
+// it has a long-running waitUntil promise.
+const WAIT_UNTIL_TIMEOUT = 10;
+export const WAIT_UNTIL_TIMEOUT_MS = 10 * 1000;
+
+export const WAIT_UNTIL_WARNING = (entrypointPath: string) =>
```

Comment:
```suggestion
export const waitUntilWarning = (entrypointPath: string) =>
```

Lets follow the naming convention for functions